static/app/types/group.tsx

```diff
@@ -52,6 +52,25 @@ export enum IssueType {
   PERFORMANCE_N_PLUS_ONE = 'performance_n_plus_one',
 }
 
+/**
+ * Defines what capabilities a category of issue has. Not all categories of
+ * issues work the same.
+ */
+export type IssueCategoryCapabilities = {
+  /**
+   * Is this issude deletable
+   */
+  delete: boolean;
+  /**
+   * Is this issude deletable
+   */
+  ignore: boolean;
+  /**
+   * Is this issue mergable?
+   */
+  merge: boolean;
+};
+
```

Comment:
```suggestion
export type IssueCategoryCapabilities = {
  /**
   * Is this issue deletable
   */
  delete: boolean;
  /**
   * Is this issue ignorable
   */
  ignore: boolean;
  /**
   * Is this issue mergeable
   */
  merge: boolean;
};

```

Reply:
wait what is different here 🤔

Reply:
typoes in the comments 😄

Reply:
`issude` typo, `deletable` where it should be `ignorable`, `mergeable` instead of `mergable`

Reply:
tyvm